aicsimageio/aics_image.py

```diff
@@ -620,14 +620,14 @@ def __repr__(self) -> str:
         return f"<AICSImage [{type(self.reader).__name__}]>"
 
     @property
-    def cluster(self) -> Optional["distributed.LocalCluster"]:
+    def cluster(self) -> Optional["distributed.LocalCluster"]:  # noqa: F821
```

Comment:
Why do you need this? It seems to have something to do with _variable definitions but I don't see that here. What am I missing?

Reply:
Why do we need what? The `noqa`? The typing?